public/styles/_docs.scss

```diff
@@ -189,3 +190,48 @@
   margin-right: auto;
   margin-left: auto;
 }
+
+.docs-guides {
+  h2 {
+    // Match the default from primer
+    margin-bottom: 12px;
+    padding-bottom: 12px;
+  }
+
+  .docs-guides-list {
+    column-count: 3;
```

Comment:
The 4 column list below this one uses classes to only switch to 4 columns at >992px wide, can we similarly add a media query around this one?

```css
@media (min-width: 922px) {
  .docs-guides-list {
    column-count: 3;
  }
}
```